source/ddbus/thin.d

```diff
@@ -16,9 +16,8 @@ import std.algorithm;
 public import ddbus.exception : wrapErrors, DBusException;
 
 struct ObjectPath {
-  private string _value;
```

Comment:
That line should stay...

Reply:
oops eh must have messed up on the copy paste there